### ISSUE TITLE
The accuracy rate is about 66% when I run keras-VGG16 model on  verification dataset. I am confused and I wonder  if  the labels is wrong.

### ISSUE THREAD

**Issue by Lxyzgf** [fchollet/keras#5583]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!
[synset_words.txt](https://github.com/fchollet/keras/files/815898/synset_words.txt)

- [ ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [ ] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by Lxyzgf** [comment]
The validation dataset is ILSVRC2012-val dataset. looking forward your reply!

**Comment by tpinetz** [comment]
You should probably read the paper first. https://arxiv.org/pdf/1409.1556/. Their they have an error rate of 28.1 which would be 71.9% accuracy on a larger image size. This is probably the accuracy they get on that dataset.

**Comment by Lxyzgf** [comment]
@tpinetz  Thank you very much! I will read it .
